### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Sophyass** [community/community#70183] [GitHub Education]
Hello,

My name is Sophia and I am trying to download the Github chat feature but it says I need to have a copilot account. I submitted the student starter pack under the name 'Sophyass' on Friday and am waiting to here back that I can have a free copilot account as a student in order to complete an assignment due this week. Can you please expedite this or let me know when I can expect to have a copilot account approved?

Thanks,
Sophia

**Comment by Anemaygi** [comment]
Hi,

If you already have your student developer pack, you  need to go to the copilot website (https://copilot.github.com/?WT.mc_id=academic-88217-leestott) and login. Then, go to settings and connect your Github account to Copilot.

**Source:** https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-setting-up-github-student-and-github-copilot-as-an/ba-p/3736279

I hope it helps!
